### ISSUE TITLE
ModuleNotFoundError: No module named 'keras.api' when using Keras 3

### ISSUE THREAD

**Issue by jedlimlx** [keras-team/keras-cv#2110]
### Current Behavior:
```
/usr/lib/python3.10/importlib/__init__.py in import_module(name, package)
    124                 break
    125             level += 1
--> 126     return _bootstrap._gcd_import(name[level:], package, level)
    127 
    128 

ModuleNotFoundError: No module named 'keras.api'
```

This error occurs when I attempt to import keras_cv when Keras 3 is installed.

### Expected Behavior:
It should import without any errors, according to this commit https://github.com/keras-team/keras-cv/commit/35bf1896e98164b95d8b34bdad448f2a28b5e54b.

### Steps To Reproduce:
https://colab.research.google.com/gist/jedlimlx/bb733ca6f4d278ef745a19e8bc22cf0f/keras-cv-bug.ipynb

Basically, just install the latest versions of Keras 3 and Keras CV, and then try and import keras_cv.

### Version:
https://github.com/keras-team/keras-cv/commit/097bebec801c9ef34ac74aed56667e416a14ec75
https://github.com/keras-team/keras/commit/26831056309d7af88899941d52ba0a7d987d7c62

**Comment by EdIzaguirre** [comment]
Don't think this is bug. Until Keras 3 officially comes out, Keras_CV is using Keras-Core. 



Check out [this Colab.](https://colab.research.google.com/drive/1324gOk82wjbiB51vfXGZ9HMBi6EUdDOr?usp=sharing)

**Comment by ianstenbit** [comment]
thanks for the issue! @EdIzaguirre is correct that for now we're using Keras Core.

Having said that, we are working on getting Keras 3 fully supported, and I hope to have the live GitHub version working with Keras 3 by next week.

**Comment by sampathweb** [comment]
Here's how you could use Keras 3 with Keras-CV and should work.  Please re-open if you still run into any issues.
```
!pip install -U tensorflow keras  # Keras-CV needs TF 2.15 when used with Keras 3
!pip install -U keras-cv
import os
os.environ["KERAS_BACKEND"] = "tensorflow"
import keras
import keras_cv
```
